data/mods/No_Rail_Stations/modinfo.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "type": "MOD_INFO",
+    "ident": "No_Rail_Stations",
+    "name": "No Rail Stations",
+    "authors": [ "Inglonias" ],
+    "description": "Removes above-ground rail stations from the game.",
+    "category": "monster_exclude",
```

Comment:
I think this is the wrong category for this mod. Anybody got a better one?

Reply:
Buildings

Reply:
That puts it into the first column of mods, rather than the exclusion section, but fine. Done.

Reply:
That mod selection menu needs rework anyway.